### ISSUE TITLE
[Bug Report] There was a bug when Select used filterable

### ISSUE THREAD

**Issue by Yiiu** [ElemeFE/element#12718]
### Element UI version
2.4.6

### OS/Browsers version
windows 7

### Vue version
2.5.17

### Reproduction Link
[https://jsfiddle.net/37swdgfb/](https://jsfiddle.net/37swdgfb/)

### Steps to reproduce
When using the filterable of the select component, the first selection of the drop-down option will always pop up twice.

### What is Expected?
Pop up only once

### What is actually happening?
Popped up twice

**Comment by ziyoung** [comment]
Duplicate of #11069
